src/service/resources-impl.js

```diff
@@ -1772,6 +1775,13 @@ export class Resources {
     if (loadRect) {
       for (let i = 0; i < this.resources_.length; i++) {
         const r = this.resources_[i];
+        // TODO(dvoytenko): This extra build has to be merged with the
+        // scheduleLayoutOrPreload_ method below.
+        if (!r.isBuilt() && !r.hasOwner() && r.hasBeenMeasured() &&
```

Comment:
Nit: Add comment like "Force build all measured, visible resources in viewport."